### PR TITLE
Adds script to generate tag message

### PR DESCRIPTION
**Pull request by conorsch**
When tagging the "securedrop-debian-packaging" repository, we include
a list of all package versions, to make reasoning about releases easier
down the road. Collecting this information manually is error-prone, so
let's provide a routine method of generating that info so it can easily
be pasted into a prompt for an annotated git tag, consistent with docs.

### CODE REVIEW

**Comment by eloquence** [comment]
Discussed in sprint planning today, can wait until the next package release.

**Comment by conorsch** [comment]
Rebased, mostly to include #198, but also because we're moving into QA, and this script will likely save maintainers some time.

**Comment by emkll** [comment]
rebased on latest `main`

**Review by emkll — APPROVED** [review]

**Pull request merged**
